### PR TITLE
Airbnb refactor

### PR DESCRIPTION
**Pull request by azdoolittle**
Sorry for this long list. Most commits on Jan 27 reflect Airbnb style changes. The rest is front end
I'm also getting this error in Chrome: 

`Warning: Failed prop type: Invalid prop `channels` of type `array` supplied to `ChannelList`, expected `object`.
    in ChannelList (created by App)
    in App`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -117,4 +117,7 @@ main_bundle.js
 
 psd
 thumb
-sketch
\ No newline at end of file
+sketch
+
+### eslint ###
+.eslintrc.js
```
Comment (Kraig5) [inline .gitignore]: Why gitignore .eslintrc?

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    super(props);
+    this.state = {};
+    this.state.channels = [];
+    console.log('state:', this.state.channels)
+  }
+
+  componentDidMount() {
+    fetch('/channels')
+      .then((response) => {
+        return response.json()
```
Comment (Kraig5) [inline client/src/components/App.jsx]: My code is missing this return, thanks!

File: server/index.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 const express = require('express');
-const app = express();
-const port = 2004;
-
-app.use(express.static('public'))
+const router = require('./router');
 
-app.get('/', (req, res) => res.sendStatus(200));
+const app = express();
+const port = 3000;
```
Comment (Kraig5) [inline server/index.js]: 3004?

**Comment by Kraig5** [comment]
I added a comment to channelList.jsx about your type mismatch error you mentioned. Not sure if it's showing up in this PR.

File: db/seed.js [hunk 4]
```diff
@@ -1,32 +1,35 @@
...
+  channel.thumbnail_url = faker.internet.avatar();
   return channel;
-}
+};
 
-createChannels = () => {
-  var channels = [];
-  for (var i = 0; i < 100; i++) {
+const createChannels = () => {
+  const channels = [];
```
Comment (priscilagonzalez) [inline db/seed.js]: I had to do airbnb refactoring as well. :)

File: db/dbHelpers.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+const Channel = require('./index');
+
+const helper = {
+  getChannels: (callback) => {
+    Channel.find({ live: { $eq: true } }, callback).sort({ viewer_count: -1 }).limit(10);
```
Comment (priscilagonzalez) [inline db/dbHelpers.js]: I didn't know you can pass `-` to `sort` to have a descending order. Nice!

Comment (azdoolittle) [reply db/dbHelpers.js]: Yeah, I learned you can do a lot to change the query. In line 5, .find() is only getting channels that have a live === true. Then I pass -1 to have the highest viewers on top of the list.

File: server/index.js (hunk 3 above)
Comment (azdoolittle) [reply server/index.js]: Oh right. It used to be a different port, but 3000 was so convenient to type for my fingers haha

File: .gitignore (hunk 1 above)
Comment (azdoolittle) [reply .gitignore]: I was thinking that .eslintrc would only be helpful for me as the author and therefore unnecessary when everyone else pulls it.

File: db/dbHelpers.js (hunk 5 above)
Comment (priscilagonzalez) [reply db/dbHelpers.js]: Nice! Lgtm.

**Pull request merged**